src/components/Select.jsx

```diff
@@ -1,5 +1,57 @@
-const Select = () => {
-  return <div>Filter</div>;
+"use client";
+
+import { BiSolidUpArrow } from "react-icons/bi";
+import { BsDot } from "react-icons/bs";
+import Dropdown from "react-bootstrap/Dropdown";
+import { useState } from "react";
+
+const Toggle = ({ onClick, show, current, placeholder }) => {
+  return (
+    <button
+      onClick={onClick}
+      className="bg-code-darkgray text-code-white w-1/6 py-2 rounded-lg flex items-center justify-between px-3"
+    >
+      {current || placeholder}
+      <BiSolidUpArrow
+        className={`${
+          show && "rotate-180"
+        } duration-300 text-code-white text-xs`}
+      />
+    </button>
+  );
+};
+
+const Select = ({ current, setCurrent, placeholder, options }) => {
+  const [show, setShow] = useState(false);
+
+  return (
+    <Dropdown
+      show={show}
+      className="w-full m-0"
+      onToggle={() => setShow(!show)}
+      autoClose={true}
+    >
+      <Dropdown.Toggle
+        show={show}
+        current={current}
+        placeholder={placeholder}
+        as={Toggle}
+      />
+      <div className="p-1" />
```

Comment:
what's the point of this?

Reply:
the figma file has a pretty big gap between the button and menu, was just following that